CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.1] 
```

Comment:
Can you also add this link below [Unreleased]?

`[1.1.1]: https://github.com/letsar/binder/compare/releases/tag/v1.1.1`